pkg/app/kafka.go

```diff
@@ -184,6 +184,11 @@ func (kc *KafkaCluster) Uninstall(ctx context.Context) error {
 		return errors.Wrap(err, "failed to create helm client")
 	}
 
+	err = kc.cli.CoreV1().ConfigMaps(kc.namespace).Delete(ctx, configMapName, metav1.DeleteOptions{})
+	if err != nil {
```

Comment:
```suggestion
	if err != nil && !apierrors.IsNotFound(err) {
```
Please add an exception for `IsNotFound` https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors#IsNotFound